File: laramie-ingresos-publicos-webapi/src/domain/entities/localidad.ts
export default class Localidad {

    id: number;
	codigo: string;
	nombre: string;
	orden: number;
	idProvincia: number;

	constructor(
        id: number = 0,
		codigo: string = "",
		nombre: string = "",
		orden: number = 0,
		idProvincia: number = 0
	)
	{
        this.id = id;
		this.codigo = codigo;
		this.nombre = nombre;
		this.orden = orden;
		this.idProvincia = idProvincia;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.codigo = row.codigo ?? "";
		this.nombre = row.nombre ?? "";
		this.orden = row.orden ?? 0;
		this.idProvincia = row.idProvincia ?? 0;
	}

}

File: modules/administracion/models/localidad.py
from sqlalchemy import Column, BigInteger, String, Integer, ForeignKey

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Localidad(Base):
    __tablename__ = "administracion_localidades"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    codigo = Column(String(50), nullable=False)
    nombre = Column(String(250), nullable=False)
    orden = Column(Integer, nullable=False, default=0)
    id_provincia = Column(BigInteger, ForeignKey("administracion_provincias.id"), nullable=False)
